packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol

```diff
@@ -129,14 +179,17 @@ contract AnchorStateRegistry is Initializable, ISemver {
     /// @param _game The game to check.
     /// @return Whether the game is of a respected game type.
     function isGameRespected(IDisputeGame _game) public view returns (bool) {
+        // We don't do a try/catch here for legacy games because by the time this code is live on
+        // mainnet, users won't be using legacy games anymore. Avoiding the try/catch simplifies
+        // the logic.
         return _game.wasRespectedGameTypeWhenCreated();
```

Comment:
There's no guarantee that the supplied game will have this method. I think the argument there is that upgrade 12 will be done first which adds the new method and users are only likely to interact with recently created games. So if users interact with older games without this method, the call will revert instead of returning false, but that's still safe, albeit less friendly.

Reply:
Since this is going into upgrade 14? Or beyond I suspect it's fine. Definitely less friendly but users do generally only interact with relatively recent dispute games which will all have this method. Once we move to Super Roots, all games that you can interact with will have this method. I don't love the try/catch here so I'd rather remove it.